### ISSUE TITLE
Cannot injector to victim 

### ISSUE THREAD

**Issue by TARI0510** [h3xduck/TripleCross#46]
I try to inject to victim through
```bash
./injector -c 192.168.192.16
```
<img width="1571" alt="image" src="https://user-images.githubusercontent.com/39155974/198209742-2954cba1-a18c-44c3-b9fc-b73bb9f3bb0f.png">

but cannot spawn a shell from victim

could I get any help from you?

**Comment by h3xduck** [comment]
It looks like the initial handshake between the backdoor and the client is not being completed correctly.
Some things you may check are whether:
1) Both VMs can communicate between themselves
2) You are first installing the rootkit with the script, then running the client
3) An initial TCP packet with payload CC_SYN is being delivered at the machine with the rootkit (using wireshark or similar)
4) A TCP packet with payload CC_ACK is being sent from the machine with the rootkit.

**Comment by TARI0510** [comment]
Thanks for your reply~
You are right, it's the initial handshake between the backdoor and the client.

1. Both VMs can communicate between themselves
<img width="1157" alt="image" src="https://user-images.githubusercontent.com/39155974/198233193-251cabd4-7311-47ba-8e47-155c509cf611.png">

attacker: 192.168.192.168
victim: 192.168.192.169

2. I try to install the rootkit through
```bash
git clone https://github.com/h3xduck/TripleCross
cd TripleCross/src
make all
cd ../client
make
cd ../helpers
./packager.sh
```
And all goes well, but I can't receive a shell from victim
<img width="1598" alt="image" src="https://user-images.githubusercontent.com/39155974/198241332-c9606086-b5e1-455b-8d88-cbc18c8a65af.png">

3. CC_SYN can send to the victim machine
<img width="1013" alt="image" src="https://user-images.githubusercontent.com/39155974/198242551-dba114f6-c539-4c36-abd0-4383488c3faa.png">

4. Attacker can't receive CC_ACK from victim machine
<img width="965" alt="image" src="https://user-images.githubusercontent.com/39155974/198242951-f7abb618-3069-4afe-90d5-4578483c4365.png">

**Comment by TARI0510** [comment]
It seems like the initial handshake is dial to 9000 port, Do I need to listen on 9000 port with `nc`?

I try to listen on 9000 port with `nc`, but I can't get any response
<img width="973" alt="image" src="https://user-images.githubusercontent.com/39155974/198247044-e21b7557-2875-4ecc-bae1-7ad4cfaf4368.png">

What may the problem?

**Comment by h3xduck** [comment]
Oh, I just realised. If you look at the README you'll see that I prepared this client mode (using -c) to only work after activating the execution hijacking module (you'll need to configure the parameters described at https://github.com/h3xduck/TripleCross#execution-hijacking-module for the attack to happen and thus start to listen for connections).

If you just want test the backdoor and spawn a shell, use the -e or -s flags, those definitely work out of the box.

**Comment by TARI0510** [comment]
Yeah! -e and -s flags can spawn a shell well~ Thanks for your patience.
I want to use -c flag to spawn a shell according to README.
<img width="1494" alt="image" src="https://user-images.githubusercontent.com/39155974/198421190-4febac4f-a3f1-406b-b57f-cac24d98c326.png">
Maybe it is my misunderstanding using, It can't work.
